Unithereum.CodeGen/UnithereumCodeGenProcessor.cs

```diff
@@ -274,40 +352,49 @@ private static string GetDefaultShell()
                 var output = process.StandardOutput.ReadToEnd();
                 process.WaitForExit();
                 const string dsclPrefix = "UserShell:";
-                if (!output.StartsWith(dsclPrefix)) throw new InvalidOperationException(
-                    "Could not get the default shell of the current user needed to find the dotnet executable"
-                    + " required for Nethereum code generation.");
+                if (!output.StartsWith(dsclPrefix))
+                {
+                    throw new InvalidOperationException(
+                        "Could not get the default shell of the current user needed to find the dotnet executable"
+                            + " required for Nethereum code generation."
+                    );
+                }
+
                 defaultShell = output[dsclPrefix.Length..].Trim();
             }
             else
             {
                 try
                 {
                     defaultShell = File.ReadLines("/etc/passwd")
-                        .First(line => line.StartsWith(Environment.UserName + ":")).Split(':')[6];
+                        .First(line => line.StartsWith(Environment.UserName + ":"))
+                        .Split(':')[6];
                 }
                 catch (InvalidOperationException)
                 {
                     throw new InvalidOperationException(
                         "Could not find the entry for the current user in /etc/passwd needed to find the dotnet"
-                        + " executable required for Nethereum code generation.");
+                            + " executable required for Nethereum code generation."
```

Comment:
I wonder if there's a way to have the concatenated string on the other line to not have an indentation?

Reply:
Also CSharpier behavior... seems not configurable 😭